### PR TITLE
Six new Unathi themed recipes

### PR DESCRIPTION
**Pull request by blackwolf5505**
Small handful of new lizard food recipes, as well as adding the ingredients to make them. AS WELL as changing the sprite for the jellyfish meat as I didn't realize it already had one when I made this. Will require addition to wiki.

Send help, I'm not a coder.
image

### CODE REVIEW

File: html/changelogs/50n00b_sintafoods.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: 50_n00b
+
+delete-after: True
+
+changes:
+	- rscadd: "Added six recipes inspired by sinta cuisine, as well as the required ingredients to make them. They are: Shreiker Wings, Jelly Stew, Roe Fritters, Stuffed Fish Fillet, Razir Noodles, and Sinta Pudding."
```
Comment (Geevies) [inline html/changelogs/50n00b_sintafoods.yml:6]:
```suggestion
  - rscadd: "Added six recipes inspired by sinta cuisine, as well as the required ingredients to make them. They are: Shreiker Wings, Jelly Stew, Roe Fritters, Stuffed Fish Fillet, Razir Noodles, and Sinta Pudding."
```

File: code/modules/reagents/reagent_containers/food/snacks/fish.dm [hunk 2]
```diff
@@ -23,6 +23,13 @@
 	name = "fish fillet"
 	desc = "A fillet of fish."
 
+/obj/item/reagent_containers/food/snacks/fish/roe
+	name = "roe sack"
+	desc = "A fleshy organ filled with fish eggs."
+	icon_state = "roesack"
+	fish_type = "fish"
+	reagents_to_add = list(/decl/reagent/nutriment/protein/seafood/ = 3)
```
Comment (Geevies) [inline code/modules/reagents/reagent_containers/food/snacks/fish.dm:31]:
```suggestion
	reagents_to_add = list(/decl/reagent/nutriment/protein/seafood = 3)
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -5184,4 +5184,66 @@
...
+	reagents_to_add = list(/decl/reagent/nutriment = 3, /decl/reagent/nutriment/protein/seafood = 8, /decl/reagent/nutriment/protein/egg = 3, /decl/reagent/hyperzine = 5, /decl/reagent/acid/polyacid = 3)
+	reagent_data = list(/decl/reagent/nutriment = list("molten heat" = 3, "slippery noodles" = 3))
+	bitesize = 10
+	trash = /obj/item/trash/plate
+
+obj/item/reagent_containers/food/snacks/sintapudding
+	name = "sinta pudding"
+	desc = "Reddish, and extremely smooth, chocolate pudding, rich in iron!"
+	icon_state = "sintapudding"
+	reagents_to_add = list(/decl/reagent/nutriment = 1, /decl/reagent/nutriment/protein/ = 1, /decl/reagent/blood = 6, /decl/reagent/nutriment/coco = 3)
```
Comment (Geevies) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5246]:
```suggestion
	reagents_to_add = list(/decl/reagent/nutriment = 1, /decl/reagent/nutriment/protein = 1, /decl/reagent/blood = 6, /decl/reagent/nutriment/coco = 3)
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 4]
```diff
@@ -5184,4 +5184,66 @@
...
+	desc = "Fried patties made from fish eggs."
+	icon_state = "fritters"
+	reagents_to_add = list(/decl/reagent/nutriment = 3, /decl/reagent/nutriment/coating/batter = 5, /decl/reagent/nutriment/protein/seafood = 6)
+	reagent_data = list(/decl/reagent/nutriment = list("brine" = 3, "fish" = 3))
+	bitesize = 6
+	trash = /obj/item/trash/plate
+
+obj/item/reagent_containers/food/snacks/stuffedfish
+	name = "stuffed fish fillet"
+	desc = "A fish fillet stuffed with small eggs and cheese"
```
Comment (Geevies) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5217]:
```suggestion
	desc = "A fish fillet stuffed with small eggs and cheese."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 5]
```diff
@@ -5184,4 +5184,66 @@
...
+	icon_state = "stuffedfish"
+	reagents_to_add = list(/decl/reagent/nutriment = 3, /decl/reagent/nutriment/protein/seafood = 7, /decl/reagent/nutriment/protein/cheese = 2)
+	reagent_data = list(/decl/reagent/nutriment = list("brine" = 3, "fish" = 3))
+	bitesize = 5
+	trash = /obj/item/trash/plate
+
+obj/item/reagent_containers/food/snacks/stuffedcarp
+	name = "stuffed fish fillet"
+	desc = "A fish fillet stuffed with small eggs and cheese"
```
Comment (Geevies) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5226]:
```suggestion
	desc = "A fish fillet stuffed with small eggs and cheese."
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 6]
```diff
@@ -92,3 +92,65 @@
...
+	/obj/item/reagent_containers/food/snacks/egg,
+	)
+	result = /obj/item/reagent_containers/food/snacks/razirnoodles
+
+
+/decl/recipe/sintapudding
+	appliance = SAUCEPAN | POT
+	reagents = list(/decl/reagent/blood = 15, /decl/reagent/sugar = 10, /decl/reagent/nutriment/coco = 5)
+	items = list(
+	/obj/item/reagent_containers/food/snacks/spreads/butter
```
Comment (Geevies) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:152]:
```suggestion
		/obj/item/reagent_containers/food/snacks/spreads/butter
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 7]
```diff
@@ -92,3 +92,65 @@
...
+	)
+	result = /obj/item/reagent_containers/food/snacks/stuffedcarp
+
+/decl/recipe/razirnoodles
+	appliance = SKILLET | SAUCEPAN
+	fruit = list("lime" = 1)
+	reagents = list(/decl/reagent/sodiumchloride = 1)
+	items = list(
+	/obj/item/reagent_containers/food/snacks/dwellermeat,
+	/obj/item/reagent_containers/food/snacks/egg,
```
Comment (Geevies) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:143]:
```suggestion
		/obj/item/reagent_containers/food/snacks/dwellermeat,
		/obj/item/reagent_containers/food/snacks/egg,
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 8]
```diff
@@ -92,3 +92,65 @@
...
+	/obj/item/reagent_containers/food/snacks/cheesewedge,
+	)
+	result = /obj/item/reagent_containers/food/snacks/stuffedfish
+
+/decl/recipe/stuffedcarp
+	appliance = SKILLET | SAUCEPAN
+	items = list(
+	/obj/item/reagent_containers/food/snacks/fish/carpmeat,
+	/obj/item/reagent_containers/food/snacks/fish/roe,
+	/obj/item/reagent_containers/food/snacks/cheesewedge,
```
Comment (Geevies) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:133]:
```suggestion
		/obj/item/reagent_containers/food/snacks/fish/carpmeat,
		/obj/item/reagent_containers/food/snacks/fish/roe,
		/obj/item/reagent_containers/food/snacks/cheesewedge,
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 9]
```diff
@@ -92,3 +92,65 @@
...
+	)
+	result = /obj/item/reagent_containers/food/snacks/jellystew
+
+
+/decl/recipe/stuffedfish
+	appliance = SKILLET | SAUCEPAN
+	items = list(
+	/obj/item/reagent_containers/food/snacks/fish/fishfillet,
+	/obj/item/reagent_containers/food/snacks/fish/roe,
+	/obj/item/reagent_containers/food/snacks/cheesewedge,
```
Comment (Geevies) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:124]:
```suggestion
		/obj/item/reagent_containers/food/snacks/fish/fishfillet,
		/obj/item/reagent_containers/food/snacks/fish/roe,
		/obj/item/reagent_containers/food/snacks/cheesewedge,
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 10]
```diff
@@ -92,3 +92,65 @@
...
+	result = /obj/item/reagent_containers/food/snacks/batwings
+
+/decl/recipe/jellystew
+	appliance = POT
+	fruit = list("chili" = 2)
+	reagents = list(/decl/reagent/water = 10)
+	items = list(
+	/obj/item/reagent_containers/food/snacks/fish/cosmozoan,
+	/obj/item/reagent_containers/food/snacks/cutlet,
+	/obj/item/reagent_containers/food/snacks/spreads/butter,
```
Comment (Geevies) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:114]:
```suggestion
		/obj/item/reagent_containers/food/snacks/fish/cosmozoan,
		/obj/item/reagent_containers/food/snacks/cutlet,
		/obj/item/reagent_containers/food/snacks/spreads/butter,
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 11]
```diff
@@ -92,3 +92,65 @@
...
 	result = /obj/item/reagent_containers/food/snacks/sliceable/sushi_roll
+
+
+/decl/recipe/batwings
+	appliance = SKILLET | SAUCEPAN
+	fruit = list("chili" = 2)
+	items = list(
+	/obj/item/reagent_containers/food/snacks/meat/bat,
+	/obj/item/reagent_containers/food/snacks/meat/bat,
+	/obj/item/reagent_containers/food/snacks/spreads/butter,
```
Comment (Geevies) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:103]:
```suggestion
		/obj/item/reagent_containers/food/snacks/meat/bat,
		/obj/item/reagent_containers/food/snacks/meat/bat,
		/obj/item/reagent_containers/food/snacks/spreads/butter,
```

**Review by desvenlafaxine — CHANGES_REQUESTED** [review]
Add the bitesizes from #12330 so that there are no more conflicts.

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 12]
```diff
@@ -92,3 +92,65 @@
...
 	result = /obj/item/reagent_containers/food/snacks/sliceable/sushi_roll
+
+
+/decl/recipe/batwings
+	appliance = SKILLET | SAUCEPAN
+	fruit = list("chili" = 2)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/meat/bat,
+		/obj/item/reagent_containers/food/snacks/meat/bat,
+		/obj/item/reagent_containers/food/snacks/spreads/butter,
```
Comment (Casper3667) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:103]:
```suggestion
		/obj/item/reagent_containers/food/snacks/spreads/butter
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 13]
```diff
@@ -92,3 +92,65 @@
...
+	result = /obj/item/reagent_containers/food/snacks/batwings
+
+/decl/recipe/jellystew
+	appliance = POT
+	fruit = list("chili" = 2)
+	reagents = list(/decl/reagent/water = 10)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/fish/cosmozoan,
+		/obj/item/reagent_containers/food/snacks/cutlet,
+		/obj/item/reagent_containers/food/snacks/spreads/butter,
```
Comment (Casper3667) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:114]:
```suggestion
		/obj/item/reagent_containers/food/snacks/spreads/butter
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 14]
```diff
@@ -92,3 +92,65 @@
...
+	)
+	result = /obj/item/reagent_containers/food/snacks/jellystew
+
+
+/decl/recipe/stuffedfish
+	appliance = SKILLET | SAUCEPAN
+	items = list(
+		/obj/item/reagent_containers/food/snacks/fish/fishfillet,
+		/obj/item/reagent_containers/food/snacks/fish/roe,
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
```
Comment (Casper3667) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:124]:
```suggestion
		/obj/item/reagent_containers/food/snacks/cheesewedge
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 15]
```diff
@@ -92,3 +92,65 @@
...
+	)
+	result = /obj/item/reagent_containers/food/snacks/stuffedfish
+
+/decl/recipe/stuffedcarp
+	appliance = SKILLET | SAUCEPAN
+	items = list(
+		/obj/item/reagent_containers/food/snacks/fish/carpmeat,
+		/obj/item/reagent_containers/food/snacks/fish/roe,
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
```
Comment (Casper3667) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:133]:
```suggestion
		/obj/item/reagent_containers/food/snacks/cheesewedge
```

File: code/modules/cooking/recipes/cultural/recipes_unathi.dm [hunk 16]
```diff
@@ -92,3 +92,65 @@
...
+	)
+	result = /obj/item/reagent_containers/food/snacks/stuffedcarp
+
+/decl/recipe/razirnoodles
+	appliance = SKILLET | SAUCEPAN
+	fruit = list("lime" = 1)
+	reagents = list(/decl/reagent/sodiumchloride = 1)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/dwellermeat,
+		/obj/item/reagent_containers/food/snacks/egg,
```
Comment (Casper3667) [inline code/modules/cooking/recipes/cultural/recipes_unathi.dm:143]:
```suggestion
		/obj/item/reagent_containers/food/snacks/egg
```

**Review by Casper3667 — APPROVED** [review]
Mostly minor formatting nitpicks.

**Comment by blackwolf5505** [comment]
!review

**Review by desvenlafaxine — COMMENTED** [review]
Another of my PRs fucked this up, this time being #12351. Please update.

**Comment by blackwolf5505** [comment]
can't figure out how to use desktop to fix this. can't be assed to babysit the PR for conflicts waiting for it to be merged. closing.

**Pull request closed without merging**